src/server/JasmineGraphInstanceService.cpp

```diff
@@ -591,19 +593,23 @@ void *instanceservicesession(void *dummyPt) {
                 std::cout << "Thread " << threadCount << " joined" << std::endl;
             }
 
-            std::vector<char *> vc;
-            std::transform(trainargs.begin(), trainargs.end(), std::back_inserter(vc), converter);
+            write(connFd, JasmineGraphInstanceProtocol::SEND_PARTITION_ITERATION.c_str(), JasmineGraphInstanceProtocol::SEND_PARTITION_ITERATION.size());
+            instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::SEND_PARTITION_ITERATION, "info");
 
-            std::string path = "cd " + utils.getJasmineGraphProperty("org.jasminegraph.graphsage") + " && ";
-            std::string command = path + "python3 -m unsupervised_train ";
+            bzero(data, 301);
```

Comment:
shall we declare these 300 and 301 values as constants?

Reply:
Done

Reply:
Ok